src/configuration/GPG/GnuPG/gpg.conf

```diff
@@ -205,8 +205,7 @@ keyserver hkp://keys.gnupg.net
 # photo-viewer "metamail -q -d -b -c %T -s 'KeyID 0x%k' -f GnuPG"
 
 
-personal-digest-preferences SHA256
```

Comment:
I too think SHA256 is currently safe. I suggested SHA512 because it is far more expensive to compute a collision for SHA512 than for a SHA256 hash. I tend to keep my mails for a while so this addresses mainly brute force attacks in the future.